### PR TITLE
Provide MetricHPEXS font download links

### LINKED ISSUE #1627: Download fonts from Typography page
Add something towards the bottom of the Typography page to allow the user to download the MetricHPEXS fonts. Include some brief guidance on when this should be used, which I think includes designers wanting it locally and products needing to be able to reference when users will typically not have internet access from their browser.

### PR DESCRIPTION
**Pull request by MikeKingdom**
#### What does this PR do?
Adds some simple guidance and font download links to the Typography page.

#### Where should the reviewer start?
typography.mdx

#### What testing has been done on this PR?
Manual clicking on the links from Typography page.

#### How should this be manually tested?
Go to the typography page and click on the links.

#### What are the relevant issues?
Fixes #1627 

#### Screenshots (if appropriate)
image

#### Should this PR be mentioned in Design System updates?
No

#### Is this change backwards compatible or is it a breaking change?
Backwards compatible

### CODE REVIEW

File: aries-site/src/pages/foundation/typography.mdx [hunk 1]
```diff
@@ -63,3 +63,24 @@ For accessibility reasons, always ensure [semantically correct heading levels](h
...
 </Example>
+
+## Fonts for offline usage
+
+In rare cases where a product needs to reference the MetricHPEXS fonts but users
+don't have internet access from their browser or maybe a designer needs the fonts
+locally, you can download the fonts for local use.
+
+MetricHPEXS-Regular [woff2](https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/MetricHPEXS-Regular.woff2),
+[woff](https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/MetricHPEXS-Regular.woff)
```
Comment (halocline) [inline aries-site/src/pages/foundation/typography.mdx:74]: Designers will want to reference .otf versions. Those files are located in the same directory, just with the .otf extension.
```suggestion
MetricHPEXS-Regular [woff2](https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/MetricHPEXS-Regular.woff2),
[woff](https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/MetricHPEXS-Regular.woff),
[otf](https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/MetricHPEXS-Regular.otf),
```

**Comment by halocline** [comment]
@MikeKingdom - Would the experience be better if there were a single "Download Fonts" button? We could zip all the files and place them under a single path if we wants that approach.

**Comment by ericsoderberghp** [comment]
Would it be possible to change the CDN to allow browsing the index URL (https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/) and just link to that?

**Comment by halocline** [comment]
> Would it be possible to change the CDN to allow browsing the index URL (https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/) and just link to that?

We can look into our options.

**Comment by MikeKingdom** [comment]
> > Would it be possible to change the CDN to allow browsing the index URL (https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/) and just link to that?
> 
> We can look into our options.

I like that idea also. I originally checked that but it was disallowed. We could also just have 3 zips of the different formats and link to those. There may be some issues with people/browsers trusting zips although I usually prefer them myself.

**Review by halocline — APPROVED** [review]

**Pull request merged**